README.md

```diff
@@ -1 +1,70 @@
-# MathMagicians
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```

Comment:
- [ ] It looks like you forgot to update the readme, it is still the default one that comes with `create react app`. Please update the readme with the Microverse template and customise it 😉